README.md

```diff
@@ -85,17 +98,17 @@ This app scaffold comes ready to submit for beta testing on iOS with Testflight
 
 ### Coming soon
 
-##### A Tasit list app
+##### Tasit list app
```

Comment:
Do you think with the "a" gone it's still clear that we won't be making a generalist app that supports all list dapps, but building a mobile app for a specific list dapp?

I think it maybe still be clear - just checking. Thanks!

Reply:
agree that the `a` provides greater specificity while a dapp project is being determined